lib/srv/sess.go

```diff
@@ -875,13 +876,11 @@ func (s *session) startExec(channel ssh.Channel, ctx *ServerContext) error {
 			events.EventNamespace:           ctx.srv.GetNamespace(),
 			events.SessionInteractive:       false,
 			events.SessionEnhancedRecording: s.hasEnhancedRecording,
-			events.SessionParticipants: []string{
-				ctx.Identity.TeleportUser,
-			},
```

Comment:
@russjones I wasn't sure why we opted for `events.SessionParticipants` over `events.EventUser`, so i replaced it.

Reply:
The Web UI uses this to display the list of participants in the session. Both scenarios: having a single or multiple participants is fine.